cheatsheets/Session_Management_Cheat_Sheet.md

```diff
@@ -334,7 +334,7 @@ Web applications can use JavaScript code to capture all the web browser tab or w
 
 ### Disable Web Browser Cross-Tab Sessions
 
-Web applications can use JavaScript code once the user has logged in and a session has been established to force the user to re-authenticate if a new web browser tab or window is opened against the same web application. The web application does not want to allow multiple web browser tabs or windows to share the same session. Therefore, the application tries to force the web browser to not share the same session ID simultaneously between them.
+Web applications can use JavaScript code once the user has logged in and a session has been established to force the user to authenticate again intentionally if a new web browser tab or window is opened against the same web application. The web application does not want to allow multiple web browser tabs or windows to share the same session. Therefore, the application tries to force the web browser to not share the same session ID simultaneously between them.
```

Comment:
what is wrong with re-authenticate?

Reply:
Well what exactly is 're-authenticate'? 

Are when you 're-authenticate' do you mean you are just 'authenticating' again now because the previous _perceived active_ authenticated 'session' has somehow expired or otherwise entered an untrusted state?

The paragraph talks to the point that, quote:

> The web application does not want to allow multiple web browser tabs or windows to share the same session.

Meaning the new tab was never authenticated to be 're' authenticated. 

In any context, re-authentication is either just authenticate or it can be confused with a 2nd factor authentication, or more problematic it an be used in place of authorization and sound _correct_, but we know better. Let's avoid a semantic reference to re-authentication and just use the proper terms 'authenticate', 'authorize', and 2fa where one might be using re-authentication.